app/views/warehouse_reports/hmis_cross_walks/index.xlsx.axlsx

```diff
@@ -17,9 +17,9 @@ wb.add_worksheet(name: 'Projects') do |sheet|
       [
```

Comment:
This report should probably ignore the confidentiality as well.  It doesn't include any client detail, and the entire point is to be able to lookup IDs from the LSA results and figure out which project or inventory record is causing trouble in the sending HMIS.

Reply:
OK, updated with https://github.com/greenriver/hmis-warehouse/pull/2267/commits/fc2f2181351dd31a68e75a43c506e854e6a670a7 and added to list of exceptions here https://github.com/greenriver/hmis-warehouse/pull/2267/commits/2a440ab62d1f688ae6efce46a0bb970f39897d59